src/DIRAC/WorkloadManagementSystem/DB/JobDB.py

```diff
@@ -1553,7 +1552,11 @@ def getAllSiteMaskStatus(self):
         siteDict = {}
         if result["OK"]:
             for site, status, lastUpdateTime, author, comment in result["Value"]:
-                siteDict[site] = status, lastUpdateTime, author, comment.decode()
+                try:
+                    comment = comment.decode()
```

Comment:
```suggestion
                    # TODO: This is only needed in DIRAC v8.0.x while moving from BLOB -> TEXT
                    comment = comment.decode()
```
I would also be tempted to do an `if isinstance(comment, bytes):` here but it's up to you.